e2e-tests/rs-shard-migration/run

```diff
@@ -48,7 +49,8 @@ function main() {
 
     simple_data_check "${cluster}" "${CLUSTER_SIZE}" 1 "-mongos"
 
-    if [[ -z "$(get_lastCommitedOpTime ${cluster} ${namespace})" ]]; then
+    if [[ -z "$(get_shard_parameter ${cluster} ${namespace} lastCommitedOpTime)" ]] \
```

Comment:
If I see correctly lastCommitedOpTime is not available always for 3.6 so it is expected to fail there: https://jira.mongodb.org/browse/SERVER-33018
This was fixed in 3.7 and higher versions.
Also I don't understand the logic to check this on rs0 for checking sharding.

Reply:
actually it's executed on ``-rs0-0``

Reply:
updated the comment to the actual reason